docs/man/borg-init.1

```diff
@@ -126,46 +129,46 @@ _
 .\" nanorst: inline-replace
 .
 .sp
-\fIMarked modes\fP are new in Borg 1.1 and are not backwards\-compatible with Borg 1.0.x.
-.sp
+How should a user choose between SHA-256 and Blake2b? The main difference is execution speed. 
 On modern Intel/AMD CPUs (except very cheap ones), AES is usually
 hardware\-accelerated.
 BLAKE2b is faster than SHA256 on Intel/AMD 64\-bit CPUs
 (except AMD Ryzen and future CPUs with SHA extensions),
-which makes \fIauthenticated\-blake2\fP faster than \fInone\fP and \fIauthenticated\fP\&.
+which makes \fIauthenticated\-blake2\fP faster than none and \fIauthenticated\fP\&.
```

Comment:
i guess "none" here meant the mode with that name. if you remove the markup, that is not clear any more.

Reply:
My understanding was that the markup indicates which modes are not backwards compatible with Borg1.0.x. Surely "none" is backwards compatible?

Reply:
"none" (when referring to a mode) must not look like normal text.

so:
- `"none"`
- **none**
- *none*

Just be consistent.

Reply:
ok. I think the `code` markup makes most sense when talking about command line arguments. I will use that consistently throughout the document.